pkg/config/config.go

```diff
@@ -34,7 +34,14 @@ func LoadConfig(path string) (*Spec, error) {
 
 	data, err := os.ReadFile(absPath)
 	if err != nil {
-		return nil, fmt.Errorf("error reading config file: %w", err)
+
+		// If the file does not exist, check if the fallback path exists
```

Comment:
I'm not sure if what I'm doing is idiomatic Go @jpmcb. Also before it was returning the exact error, now I mention it failed reading file x or y. Should I still error out with the actual error like before?